docs/docs/tutorials/contract_tutorials/token_contract.md

```diff
@@ -387,9 +427,9 @@ This function is called from [`burn`](#burn). The account's private balance is d
 
 #include_code reduce_total_supply /noir-projects/noir-contracts/contracts/token_contract/src/main.nr rust
 
-### Unconstrained function implementations
+### View function implementations
 
-Unconstrained functions are similar to `view` functions in Solidity in that they only return information from the contract storage or compute and return data without modifying contract storage.
+View functions in Aztec are similar to `view` functions in Solidity in that they only return information from the contract storage or compute and return data without modifying contract storage. These functions are different from unconstrained functions in that the return values are constrained by their definition in the contract.
```

Comment:
Would it make sense to have a line here about what information what is visible to the network when calling a view function? Or considerations between public/private view calls (if any?)
A view function is harmless in Ethereum (public info), but reading public info might leak something about private functions called, even if just `view`?

Reply:
just added another line. lmk what you think